pages/manager/SubscriptionHandler.inc.php

```diff
@@ -178,8 +178,20 @@ function subscriptionTypes($args, $request) {
 		$this->setupTemplate($request);
 
 		$templateMgr = TemplateManager::getManager($request);
-		$templateMgr->addJavaScript('lib/pkp/js/lib/jquery/plugins/jquery.tablednd.js');
-		$templateMgr->addJavaScript('lib/pkp/js/functions/tablednd.js');
+		$templateMgr->addJavaScript(
```

Comment:
The tablednd stuff is long since removed but this reference was orphaned. It can be removed.